src/models/classes/factoryorder.js

```diff
@@ -0,0 +1,10 @@
+class FactoryOrder extends OrderCore
+{
+    constructor(products)
+    {
+        var time = GAME.model.config.orderTransportDuration;
```

Comment:
Why? Makes much more sense to leave it in the factory submodel right? It has nothing to do with the customer.